src/viking.js

```diff
@@ -1,19 +1,43 @@
 // Soldier
-class Soldier {}
-
-// Viking
-class Viking {}
-
-// Saxon
-class Saxon {}
-
-// War
-class War {}
-
-
+class Soldier {
+  constructor (salud, fuerza) {
+    this.health = salud 
+    this.strenght = fuerza
+  }
+  attack() {
+    return this.strenght
+  }
+  receiveDamage(damage) {
+    this.health - damage
+  }
+}
 
-// The following is required to make unit tests work.
-/* Environment setup. Do not modify the below code. */
-if (typeof module !== 'undefined') {
-  module.exports = { Soldier, Viking, Saxon, War };
+//viking
+class Viking extends Soldier {
+  constructor(nombre, salud, fuerza) {
+    super(salud, fuerza)
+    this.name = nombre
+  }
+  receiveDamage(damage) {
+    this.health = this.health - damage
+    if (this.health > 0) {
+      return `${this.name} has received ${damage} points of damage`
+    } else { 
+      return `${this.name} has died in act of combat`
+    }
+  }
+  battleCry() {
+  return "Odin Owns You All"
+}
 }
+      // Saxon 
+      class Saxon extends Soldier {
```

Comment:
Solo nos falto agregar metodo de ataque 
       attack(){
    return this.strength